_data_weighting.tex

```diff
@@ -90,7 +90,7 @@ \subsection{Data Weighting}
 
 \item The input-sample-size provided by the user is an upper bound on
 weighting for those data, such that a dummy value of 1 will cause those
-data to never be assigned a weight \>1.
```

Comment:
The greater than sign was appropriate but the `\>` notation, presumably intended to include the symbol within latex, was either not needed or not working correctly so it got left off entirely in the resulting manual. I would avoid the issue and make it more clear by changing to text: `never be assigned a weight greater than 1.`